util/helm/index.go

```diff
@@ -1,59 +1,17 @@
 package helm
 
 import (
-	"errors"
 	"fmt"
-	"time"
-
-	log "github.com/sirupsen/logrus"
-
-	"github.com/Masterminds/semver/v3"
 )
 
-type Entry struct {
-	Version string
-	Created time.Time
-}
-
 type Index struct {
-	Entries map[string]Entries
+	Entries map[string][]string
 }
 
-func (i *Index) GetEntries(chart string) (Entries, error) {
-	entries, ok := i.Entries[chart]
+func (i *Index) GetTags(chart string) ([]string, error) {
```

Comment:
This means that for this case we would need to revert `GetTags` back to `GetEntries`.